src/test/PoolSwapTest.sol

```diff
@@ -58,12 +61,16 @@ contract PoolSwapTest is Test, PoolTestBase {
         (,, uint256 reserveAfter0, int256 deltaAfter0) = _fetchBalances(data.key.currency0, data.sender);
         (,, uint256 reserveAfter1, int256 deltaAfter1) = _fetchBalances(data.key.currency1, data.sender);
 
-        // Make sure youve added liquidity to the test pool!
-        if (BalanceDelta.unwrap(delta) == 0) revert NoSwapOccurred();
```

Comment:
Ah nice - yeah I think also this isn't true for example when you swap on a pool whos liquidity is entirely in one token. Edge case.. but good to remove imo

Reply:
and now its necessary for no op :P